From cef74cb5e0aee889476e74c7fef954820592452e Mon Sep 17 00:00:00 2001
From: Deborah Brouwer <deborah.brouwer@collabora.com>
Date: Thu, 23 Nov 2023 14:08:41 -0800
Subject: [PATCH 113/214] media: wave5: add OF and V4L_MEM2MEM_DRIVERS
 dependencies

Fix compile warning when CONFIG_OF=n:

drivers/media/platform/chips-media/wave5/wave5-vpu.c:274:34: warning: 'wave5_dt_ids' defined but not used [-Wunused-const-variable=]
  274 | static const struct of_device_id wave5_dt_ids[] = {
      |

Signed-off-by: Deborah Brouwer <deborah.brouwer@collabora.com>
Signed-off-by: Hans Verkuil <hverkuil-cisco@xs4all.nl>
[hverkuil: added commit log text]
---
 drivers/media/platform/chips-media/wave5/Kconfig | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/drivers/media/platform/chips-media/wave5/Kconfig b/drivers/media/platform/chips-media/wave5/Kconfig
index a3b949356cd5..77e7ae5c8f35 100644
--- a/drivers/media/platform/chips-media/wave5/Kconfig
+++ b/drivers/media/platform/chips-media/wave5/Kconfig
@@ -1,7 +1,8 @@
 # SPDX-License-Identifier: GPL-2.0
 config VIDEO_WAVE_VPU
 	tristate "Chips&Media Wave Codec Driver"
-	depends on VIDEO_DEV
+	depends on V4L_MEM2MEM_DRIVERS
+	depends on VIDEO_DEV && OF
 	select VIDEOBUF2_DMA_CONTIG
 	select VIDEOBUF2_VMALLOC
 	select V4L2_MEM2MEM_DEV
-- 
2.54.0

